Ninano/Ninano/Controllers/EventTab/SearchResultViewController.swift

```diff
@@ -66,47 +71,39 @@ final class SearchResultViewController: UIViewController {
         present(controller, animated: true, completion: nil)
     }
     
-    private func clickedLocalButton(local: LocationType) {
-        selectedLocal = local
-    }
-    
-    private func clickedFilterButton(_ sender: UIButton, _ filter: Filter) {
+    private func clickedFilterButtonColorChange(_ sender: UIButton) {
         let button = sender
-        if filters.contains(filter) {
-            if let index = filters.firstIndex(of: filter) {
-                filters.remove(at: index)
-            }
-            button.configuration?.baseBackgroundColor = .systemGray5
-            button.configuration?.cornerStyle = .capsule
-        } else {
-            filters.append(filter)
             button.configuration?.baseBackgroundColor = UIColor(hex: "D5DCF8")
             button.configuration?.cornerStyle = .capsule
-        }
     }
     
-    @IBAction func keywordNotificationButton(_ sender: UIButton) {
+    func addDate(date: String) {
+        dateFilterButton.setTitle(date, for: .normal)
+        clickedFilterButtonColorChange(dateFilterButton)
     }
     
-    @IBAction func categoryFilterButton(_ sender: UIButton) {
-        clickedFilterButton(sender, Filter.category)
+    @IBAction func keywordNotificationButton(_ sender: UIButton) {
```

Comment:
```suggestion
    @IBAction private func keywordNotificationButton(_ sender: UIButton) {
```

@IBAction 은 private로 구현하는 경우가 많죠 ✨

Reply:
private으로 구현하는 게 좋은 이유를 공부 해놓고 적용을 안했네요 ㅠㅠ 
영,,, 알려주셔서 감사합니다 !🥹 다음부터는 꼭 ! 한번 더 체크하겠습니다 !